src/php/Evaluator/MethodNodeEvaluator.php

```diff
@@ -72,7 +74,20 @@ public function evaluate(Node $node, $context)
         $call = \strtolower($call);
 
         $handler = $this->getHandler($call);
-        return call_user_func_array([$handler, $call], $args);
+
+        $result = call_user_func_array([$handler, $call], $args);
+
+        if (isset($node['chain'])) {
+            foreach ($node['chain'] as $chain) {
+                if ($result instanceof Chainable) {
+                    $result = $result->chain($chain);
+                } else {
+                    throw new MethodNodeException("Must chain on Chainable object, got " . \gettype($result));
```

Comment:
On PHP 5.6+ I believe you can import gettype with a use statement, so you don't have to refer to it by it's absolute namespace.